packages/dds/tree/src/feature-libraries/flex-tree/treeEvents.ts

```diff
@@ -48,4 +49,37 @@ export interface FlexTreeNodeEvents {
 	 * @returns a visitor to traverse the subtree or `void`.
 	 */
 	subtreeChanging(upPath: UpPath): PathVisitor | void;
```

Comment:
throw if you subscribe to these on a map tree node and then fail in the forwarding event emitter shim